src/markdown-pages/install-with-kurl/ipv6.md

```diff
@@ -15,29 +15,33 @@ sudo bash install.sh ipv6
 This example shows a valid spec for ipv6.
 
 ```
-apiVersion: cluster.kurl.sh/v1beta1
-kind: Installer
-metadata:
-  name: ipv6
-spec:
+apiVersion: "cluster.kurl.sh/v1beta1"
+kind: "Installer"
+metadata: 
+  name: "latest"
```

Comment:
@JustinN-Replicated do you want to change the name back to ipv6 here?

Reply:
@emosbaugh @JustinN-Replicated I'm ok with keeping Justin's changes to IPv6.

Reply:
Yep, sorry about that. Will adjust now